eth2near/eth2near-block-relay/index.js

```diff
@@ -168,8 +403,41 @@ class Eth2NearRelay {
   }
 }
 
+async function runEth2NearRelay () {
+  const masterAccount = RainbowConfig.getParam('near-master-account')
+  const masterSk = RainbowConfig.getParam('near-master-sk')
+  const keyStore = new nearAPI.keyStores.InMemoryKeyStore()
+  await keyStore.setKey(
+    RainbowConfig.getParam('near-network-id'),
+    masterAccount,
+    nearAPI.KeyPair.fromString(masterSk)
+  )
+  const near = await nearAPI.connect({
+    nodeUrl: RainbowConfig.getParam('near-node-url'),
+    networkId: RainbowConfig.getParam('near-network-id'),
+    masterAccount: masterAccount,
+    deps: {
+      keyStore: keyStore
+    }
+  })
+
+  const relay = new Eth2NearRelay()
+  const clientContract = new EthOnNearClientContract(
+    new nearAPI.Account(near.connection, masterAccount),
+    RainbowConfig.getParam('near-client-account')
+  )
+  await clientContract.accessKeyInit()
+  console.log('Initializing eth2near-relay...')
+  relay.initialize(clientContract, RainbowConfig.getParam('eth-node-url'))
+  console.log('Starting eth2near-relay...')
+  relay.run()
+}
+
 exports.Eth2NearRelay = Eth2NearRelay
 exports.EthProofExtractor = EthProofExtractor
 exports.ethashproof = ethashproof
 exports.EthOnNearClientContract = EthOnNearClientContract
 exports.EthOnNearProverContract = EthOnNearProverContract
+exports.runEth2NearRelay = runEth2NearRelay
+
+require('make-runnable')
```

Comment:
Why require at the end of the file?

Reply:
It's how `make-runnable` works. :)
It can only make functions runnable that are exported above.